Tests/ComposableTests.swift

```diff
@@ -210,4 +210,26 @@ class ComposableTests: XCTestCase {
         XCTAssertEqual(["a", "b", "c"].joined() as NSAttributedString, "abc".styled(with: StringStyle()))
     }
 
+    func testKerningStrippingOnLastCharacter() {
+        let styleWithTracking = StringStyle(.tracking(.point(0.5)))
+
+        let testCases: [String] = [
+            "abc",
+            "abc🍕",
+            "🍕🍕",
+        ]
+
+        for testCase in testCases {
+            let styledString = testCase.styled(with: styleWithTracking)
+
+            var range = NSRange(location: 0, length: 0)
+            let maxRange = NSRange(location: 0, length: styledString.length)
+
+            let kerning = styledString.attribute(.kern, at: 0, longestEffectiveRange: &range, in: maxRange) as? Float
+
+            let lastCharacterUnicodeLength = testCase.suffix(1).utf16.count
+            XCTAssertEqual(kerning, 0.5)
```

Comment:
When testing floating-point equality, we should be using the `accuracy:` parameter to avoid floating point precision issues. You can see it in a few places in the repo. There’s no hard and fast rule for how small to make the accuracy value, but for kerning, I’d say 0.001 is probably sufficient. Maybe 0.0001 just to be safe.